lib/resque/plugins/datadog.rb

```diff
@@ -4,25 +4,35 @@ module Datadog
       mattr_accessor :statsd
       mattr_accessor :service_name
 
+      def after_enqueue_datadog(*args)
+        return unless statsd
+        @enqueued_at = Time.now
+        statsd.event('Job Enqueued', enqueued_at.to_s, tags: job_tags(args))
```

Comment:
Should `enqueued_at.to_s` be `@enqueued_at.to_s`?

Reply:
Yep, good catch!